python/cuxfilter/charts/datashader/custom_extensions/graph_inspect_widget.py

```diff
@@ -0,0 +1,58 @@
+from bokeh.core.properties import Image, Bool, String
+from bokeh.models import Tool
+from bokeh.util.compiler import TypeScript
+
+TS_CODE = """
+import {InspectTool, InspectToolView} from
+                    "models/tools/inspectors/inspect_tool"
+import * as p from "core/properties"
+
+export class CustomInspectToolView extends InspectToolView {
+  model: CustomInspectTool
+  connect_signals(): void {
+      super.connect_signals()
+
+      this.connect(this.model.properties.active.change, () => {
+          this.model._active = this.model.properties.active.spec.value
+      })
+  }
+}
+
+export namespace CustomInspectTool {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = InspectTool.Props & {
+    _active: p.Property<Boolean>
```

Comment:
Interesting, I guess this is this because `active` is not exposed? Now that `readonly` properties area possible I think we could make this accessible on the Python side. Please feel free to open an issue!

Reply:
ya, that was precisely the reason. I'll raise on issue, thank you :)